.gitignore

```
@@ -3,4 +3,10 @@ public/dist
 infrastructure
 .env
 coverage
-.idea/
+.idea
+
+cypress/
+
+debug.log
+
+cypress.json
```

```suggestion
.idea
cypress/
debug.log
cypress.json
```

Should we be excluding `cypress.json`? Or is that a place we can store shared config, and we each use `cypress.env.json` for environment-specific config?

---

My cypress doesnt pick up the cypress.env.json file, needs to be cypress.json